lib/proxy_manager.js

```diff
@@ -160,20 +160,34 @@ function loadConfigFiles(cb) {
     stop_me[proxy_config_file] = proxy;
   });
 
+  // TODO (@theopak): Consider rewriting this to make it recursive.
   fs.readdir(CONFIG_DIR, function(err, files) {
     /* istanbul ignore if */
     if (err) return cb(err);
 
+    // Require a '.json' or '.xml' file ending and reject dotfiles
+    // TODO (@theopak): Write unit tests for this.
```

Comment:
The main change in this PR is to allow non-config files (e.g., "readme.md") in the $OAUTH_REVERSE_PROXY_CONFIG_DIR. I have yet to write unit tests although I have anecdotal evidence that the change works as expected.